### PR TITLE
add support for customSchema in resource sidebar for helm charts

### PR DESCRIPTION
**Pull request by sahil143**
**Fixes**: https://issues.redhat.com/browse/ODC-5041

**Analysis / Root cause**: missing schema sidebar for Helm charts install form

**Solution Description**: added support for custom schema in the sidebar

**Screen shots / Gifs for design review**: 

Screenshot 2020-11-09 at 12 07 58 PM
Screenshot 2020-11-09 at 12 08 14 PM

**Unit test coverage report**:

**Test setup:**

**Browser conformance**: 

- [X] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

**Review by rohitkrai03 — APPROVED** [review]
/approve

Tried it locally. Works as expected.

**Review by rohitkrai03 — CHANGES_REQUESTED** [review]
You also need to pass `schema` prop on line 128 as it is a required prop of ResourceSchema component. `<ResourceSchema kindObj={kindObj} schema={schema} />`

**Review by rohitkrai03 — APPROVED** [review]
/lgtm
/approve

**Comment by rohitkrai03** [comment]
/retitle Bug 1886861: add support for customSchema in resource sidebar for helm charts

File: frontend/packages/console-shared/src/components/formik-fields/field-types.ts [hunk 1]
```diff
@@ -87,6 +87,8 @@ export interface YAMLEditorFieldProps extends FieldProps {
   onChange?: (value: string) => void;
   onSave?: () => void;
   schemaModel?: K8sKind;
+  schemaLabel?: string;
+  schema?: any;
```
Comment (andrewballantyne) [inline frontend/packages/console-shared/src/components/formik-fields/field-types.ts:91]: @sahil143 please avoid `any` types at any and all changes you get. I think this is supposed to be `JSONSchema6` based on the singular usage here of `formSchema` in `frontend/packages/dev-console/src/components/helm/form/HelmInstallUpgradeForm.tsx`... please confirm.

cc @rohitkrai03

**Pull request merged**